demonstrations/tutorial_qchem_external.py

```diff
@@ -0,0 +1,153 @@
+r"""
+
+Using PennyLane with PySCF and OpenFermion
+==========================================
+
+.. meta::
+    :property="og:description": Learn how to integrate external quantum chemistry libraries with PennyLane.
+    :property="og:image": https://pennylane.ai/qml/_images/thumbnail_tutorial_external_libs.svg
+
+.. related::
+    tutorial_quantum_chemistry Quantum chemistry with PennyLane
+    tutorial_vqe A brief overview of VQE
+    tutorial_givens_rotations Givens rotations for quantum chemistry
+    tutorial_adaptive_circuits Adaptive circuits for quantum chemistry
+
+*Author: Soran Jahangiri — Posted: 30 December 2022. Last updated: 30 December 2022.*
+
+The quantum chemistry module in PennyLane, :mod:`qml.qchem  <pennylane.qchem>`, provides built-in
+methods for computing molecular integrals, solving Hartree-Fock equations, and constructing
+`fully-differentiable <https://pennylane.ai/qml/demos/tutorial_differentiable_HF.html>`_ molecular
+Hamiltonians. PennyLane also allows using external libraries to perform these tasks and integrate
+the results within a desired workflow. In this tutorial, you will learn how to integrate PennyLane
+with `PySCF <https://github.com/sunqm/pyscf>`_ and
+`OpenFermion <https://github.com/quantumlib/OpenFermion>`_ to compute molecular integrals and
+construct molecular Hamiltonians.
+
+Building molecular Hamiltonians
+-------------------------------
+In PennyLane, Hamiltonians for quantum chemistry are built with the
+:func:`~.pennylane.qchem.molecular_hamiltonian` function by specifying a backend for solving the
+Hartree-Fock equations. The default backend is the differentiable Hartree-Fock solver of the
+:mod:`qml.qchem <pennylane.qchem>` module. A Molecular Hamiltonian can also be constructed with a
+non-differentiable backend that uses the
+`OpenFermion-PySCF <https://github.com/quantumlib/OpenFermion-PySCF>`_ plugin, interfaced with the
+electronic structure package `PySCF <https://github.com/sunqm/pyscf>`_. This
+backend can be selected by setting ``method='pyscf'`` in
+:func:`~.pennylane.qchem.molecular_hamiltonian`. This requires the ``OpenFermion-PySCF``
+plugin to be installed by the user with

```

Comment:
```suggestion
plugin to be installed by the user with the following:
```
Adding structure, since we're not ending the sentence with a period.